src/coreclr/vm/qcallentrypoints.cpp

```diff
@@ -0,0 +1,324 @@
+    DllImportEntry(TypeBuilder_DefineProperty)
+    DllImportEntry(TypeBuilder_DefineEvent)
+    DllImportEntry(TypeBuilder_DefineMethodSemantics)
+    DllImportEntry(TypeBuilder_SetMethodImpl)
+    DllImportEntry(TypeBuilder_DefineMethodImpl)
+    DllImportEntry(TypeBuilder_GetTokenFromSig)
+    DllImportEntry(TypeBuilder_SetFieldLayoutOffset)
+    DllImportEntry(TypeBuilder_SetClassLayout)
+    DllImportEntry(TypeBuilder_SetParamInfo)
+    DllImportEntry(TypeBuilder_SetPInvokeData)
+    DllImportEntry(TypeBuilder_SetConstantValue)
+    DllImportEntry(TypeBuilder_DefineCustomAttribute)
+    DllImportEntry(MdUtf8String_EqualsCaseInsensitive)
+    DllImportEntry(MdUtf8String_HashCaseInsensitive)
+    DllImportEntry(TypeName_ReleaseTypeNameParser)
+    DllImportEntry(TypeName_CreateTypeNameParser)
+    DllImportEntry(TypeName_GetNames)
+    DllImportEntry(TypeName_GetTypeArguments)
+    DllImportEntry(TypeName_GetModifiers)
+    DllImportEntry(TypeName_GetAssemblyName)
+    DllImportEntry(AssemblyNative_GetFullName)
+    DllImportEntry(AssemblyNative_GetLocation)
+    DllImportEntry(AssemblyNative_GetResource)
+    DllImportEntry(AssemblyNative_GetCodeBase)
+    DllImportEntry(AssemblyNative_GetFlags)
+    DllImportEntry(AssemblyNative_GetHashAlgorithm)
+    DllImportEntry(AssemblyNative_GetLocale)
+    DllImportEntry(AssemblyNative_GetPublicKey)
+    DllImportEntry(AssemblyNative_GetSimpleName)
+    DllImportEntry(AssemblyNative_GetVersion)
+    DllImportEntry(AssemblyNative_InternalLoad)
+    DllImportEntry(AssemblyNative_GetType)
+    DllImportEntry(AssemblyNative_GetForwardedType)
+    DllImportEntry(AssemblyNative_GetManifestResourceInfo)
+    DllImportEntry(AssemblyNative_GetModules)
+    DllImportEntry(AssemblyNative_GetModule)
+    DllImportEntry(AssemblyNative_GetExportedTypes)
+    DllImportEntry(AssemblyNative_GetEntryPoint)
+    DllImportEntry(AssemblyNative_GetImageRuntimeVersion)
+    DllImportEntry(AssemblyNative_GetIsCollectible)
+    DllImportEntry(AssemblyNative_InternalTryGetRawMetadata)
+    DllImportEntry(AssemblyNative_ApplyUpdate)
+    DllImportEntry(AssemblyNative_IsApplyUpdateSupported)
+    DllImportEntry(AssemblyNative_InitializeAssemblyLoadContext)
+    DllImportEntry(AssemblyNative_PrepareForAssemblyLoadContextRelease)
+    DllImportEntry(AssemblyNative_LoadFromPath)
+    DllImportEntry(AssemblyNative_LoadFromStream)
+#ifdef TARGET_WINDOWS
+    DllImportEntry(AssemblyNative_LoadFromInMemoryModule)
+#endif
+    DllImportEntry(AssemblyNative_GetLoadContextForAssembly)
+    DllImportEntry(AssemblyNative_TraceResolvingHandlerInvoked)
+    DllImportEntry(AssemblyNative_TraceAssemblyResolveHandlerInvoked)
+    DllImportEntry(AssemblyNative_TraceAssemblyLoadFromResolveHandlerInvoked)
+    DllImportEntry(AssemblyNative_TraceSatelliteSubdirectoryPathProbed)
+    DllImportEntry(AssemblyNative_GetAssemblyCount)
+    DllImportEntry(AssemblyNative_GetEntryAssembly)
+    DllImportEntry(AssemblyNative_GetExecutingAssembly)
```

Comment:
No particular reason. I just did most of this change with a mechanical find-replace, so I didn't want to go back and update these manually since some of these classes (mainly AssemblyNative) are used by multiple managed types.

Reply:
I'll leave these names as-is for now. We can clean this up in a future PR.